packages/core/src/theme/dhis2.mui3.theme.js

```diff
@@ -0,0 +1,158 @@
+/*
+For use with Material UI version 3+
+
+-----
+
+USAGE:
+
+import { createMuiTheme, MuiThemeProvider } from '@material-ui/core/styles'
+import { mui3theme as dhis2theme } from '@dhis2/core'
+// OR: import { theme as dhis2theme } from '@dhis2/core/theme/dhis2.mui3.theme'
+
+const MyApp = (
+    <MuiThemeProvider theme={createMuiTheme(dhis2theme)}>
+        ...<MyAppComponents />...
+    </MuiThemeProvider
+)
+
+-----
+
+To override (for development purposes only):
+
+import { theme as dhis2theme } from '@dhis2/d2-ui-core/theme/dhis2.mui3.theme'
+const customTheme = {
+  ...dhis2theme,
+  overrides: {
+    ...dhis2theme.overrides,
+    MuiToolbar: { ... },
+  },
+  ...etc
+}
+*/
+
+// Color palette from https://projects.invisionapp.com/share/A7LT4TJYETS#/screens/302550228_Color
+export const colors = {
+    accentPrimary: '#1976d2',
+    accentPrimaryDark: '#004BA0',
+    accentPrimaryLight: '#63A4FF',
+    accentPrimaryLightest: '#EAF4FF',
+
+    accentSecondary: '#00796B',
+    accentSecondaryDark: '#004C40',
+    accentSecondaryLight: '#48A999',
+    accentSecondaryLightest: '#B2DFDB',
+
+    black: '#000000',
+    greyBlack: '#494949',
+    grey: '#9E9E9E',
+    greyLight: '#E0E0E0',
+    blueGrey: '#ECEFF1',
+    snow: '#F4F6F8',
+    white: '#FFFFFF',
+
+    negative: '#E53935',
+    warning: '#F19C02',
+    positive: '#3D9305',
+    info: '#EAF4FF',
+};
+
+export const palette = {
+    common: {
+        white: colors.white,
+        black: colors.black,
+    },
+    action: {
+        active: colors.greyBlack,
+        disabled: colors.greyLight,
+    },
+    text: {
+        primary: colors.black,
+        secondary: colors.greyBlack,
+        disabled: colors.grey,
+        hint: colors.grey,
+    },
+    primary: {
+        main: colors.accentPrimary,
+        dark: colors.accentPrimaryDark,
+        light: colors.accentPrimaryLight,
+        lightest: colors.accentPrimaryLightest, // Custom extension, not used by default
+        // contrastText: 'white',
+    },
+    secondary: {
+        main: colors.accentSecondary,
+        dark: colors.accentSecondaryDark,
+        light: colors.accentSecondaryLight,
+        lightest: colors.accentSecondaryLightest, // Custom extension, not used by default
+        // contrastText: 'white',
```

Comment:
Do we need to keep the commented-out lines?

Reply:
Technically no, I left them in so that we knew they could be set there (it's the only omitted key, white is the default).  Can remove them if you think it's too messy though.